here-naksha-lib-psql/src/jvmMain/resources/naksha_sql_fixme.sql

```diff
@@ -0,0 +1,568 @@
+-- noinspection SqlResolveForFile @ routine/"ST_GeomFromTWKB"
```

Comment:
What's the purpose of this file? Do we plan to use it? If so, maybe it's worth adding some ticket and referencig it here?

Reply:
Its a backup for now, in the case we need something from it. We can delete it, when we make the first release of `3.0.0` that is not alpha or beta.